libs/features/mas/src/variants/plans.css.js

```diff
@@ -3,13 +3,19 @@ export const CSS = `
 :root {
     --consonant-merch-card-plans-width: 300px;
     --consonant-merch-card-plans-icon-size: 40px;
+    --consonant-merch-card-plans-students-width: 568px;
 }
 
 merch-card[variant="plans"] {
     --consonant-merch-card-callout-icon-size: 18px;
     width: var(--consonant-merch-card-plans-width);
 }
 
+merch-card[variant="plans"][size="students"] {
```

Comment:
please do not use `size` field for this. You need to add a new attribute like `customer-segment`

Reply:
Customer segment? You probably mean Market segment (COM / EDU)?
Customer segment is INDIVIDUAL / TEAM, that would be probably needed for business cards.

So @yesil  we will need another field in odin for market segment and new dropdown in card editor with values for COM and EDU, right?

You ruined mine and @3ch023's idea to make this extremely simple. :)

Reply:
@bozojovicic  sorry, `width="students"`
we can also create plans-edu variant, and re-use the same plans variant code without having to add a new field.
cc: @3ch023

Reply:
we can, but isn't it too much code to just have a slightly different card width?

Reply:
+ we don't add a new field, just a new 'width' variation

Reply:
I created new variant "Plans students" for this card. It inherits Plans variant completely and adds new attribute segment="students" to the card so we can add styles for this card (which is for now only the card width) and reuse all styles of the plans card. It also allows us to hide some fields in the card editor which are not needed for Students card (What's included, Quantity selector).

<img width="201" alt="Screenshot 2025-04-11 at 14 47 16" src="https://github.com/user-attachments/assets/98172c0a-6e71-4da6-a586-5386873e2e45" />